lib/expand.js

```diff
@@ -420,6 +424,9 @@ async function _expandObject({
   const nests = [];
   let unexpandedValue;
 
+  // add expansion map to the processing options
+  options = {...options, expansionMap};
```

Comment:
Since expansionMap is passed to `options` from top level `api.expand`, here might be redundant

Reply:
Reviewing this again, I agree, currently the only calls to _expandObject are coming from expand however if there are other callers of this function in future it might not hurt to have this extra layer of protection.